packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.js

```diff
@@ -330,6 +332,26 @@ export default class QuanticCategoryFacet extends LightningElement {
     this.dispatchEvent(renderFacetEvent);
   }
 
+  validateDependsOnProperty() {
+    if (this.dependsOn) {
+      getBueno(this).then(() => {
+        const {parentFacetId, expectedValue} = this.dependsOn;
+        if (!parentFacetId || !Bueno.isString(parentFacetId)) {
+          console.error(
+            `The ${this.field} ${this.template.host.localName} requires depends.parentFacetId to be a valid string.`
```

Comment:
What's `this.template.host.localName` ? the Component name?

Reply:
Also I am guessing you wanted to write this?


```suggestion
            `The ${this.field} ${this.template.host.localName} requires dependsOn.parentFacetId to be a valid string.`
```

Reply:
yes `this.template.host.localName` is the component name.